octoprint_octobullet/__init__.py

```diff
@@ -45,7 +145,7 @@ def on_settings_load(self):
 		data = octoprint.plugin.SettingsPlugin.on_settings_load(self)
 
 		# only return our restricted settings to admin users - this is only needed for OctoPrint <= 1.2.16
-		restricted = ("access_token", "push_channel")
+		restricted = ("quiet_minutes", "access_token", "push_channel")
```

Comment:
I think we don't need to restrict access to "quiet_minutes", it's not really sensitive information, right?